### PR TITLE
chore(lint-staged): update readme with lint-staged instructions

### PR DESCRIPTION
**Pull request by goldcaddy77**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,32 @@ Options:
     output usage information
 ```
 
+### Usage with pre-commit Hooks
+
+Using [lint-staged](https://github.com/okonet/lint-staged) and [husky](https://github.com/typicode/husky), you can lint
+your staged GraphQL schema file before you commmit.  First, install these packages:
```
Comment (cjoudrey) [inline README.md]: `commmit` ▶️ `commit`

File: README.md [hunk 2]
```diff
@@ -53,6 +53,32 @@ Options:
...
+```bash
+yarn add --dev lint-staged husky
+```
+
+Then add a `precommit` script and a `lint-staged` key to your `package.json` like so:
+
+```json
+{
+  "scripts": {
+    "precommit": "graphql-schema-linter"
```
Comment (cjoudrey) [inline README.md]: Do they not need to specify the path here too? i.e. `**/*.graphql`?

File: README.md [hunk 3]
```diff
@@ -53,6 +53,32 @@ Options:
...
+  },
+  "lint-staged": {
+    "*.graphql": ["graphql-schema-linter **/*.graphql"]
+  }
+}
+```
+
+The above configuration assumes that you have either one `schema.graphql` file or multiple files that should be linted
+concatenated together.  If you have both client and server schema in the same project, you'll likely need to put
+multiple entries in the `lint-staged` object above - one for client and one for server.
```
Comment (cjoudrey) [inline README.md]: They'd also have to change the path `**/*.graphql`, right?

File: README.md (hunk 2 above)
Comment (goldcaddy77) [reply README.md]: Whoa, late night coding!  This should be `lint-staged`. :)

File: README.md (hunk 3 above)
Comment (goldcaddy77) [reply README.md]: Yeah, will add a note. 👍

**Comment by cjoudrey** [comment]
:+1: :+1: Thanks again! 😀

**Pull request merged**